docs/content/latest/explore/colocated-tables/linux.md

```diff
@@ -30,12 +30,19 @@ showAsideToc: true
 
 </ul>
 
-Colocated tables can dramatically increase the number of relations (tables, indexes, etc) that can be supported per node while keeping the number of tablets per node low.
-
-In workloads that do very little IOPS and have a small data set, the bottleneck shifts from CPU/disk/network to the number of tablets one can host per node. There are practical limitations to the number of tablets that YugabyteDB can handle per node, even though this number could be very high, depending on the workload pattern. Since each table by default requires at least one tablet without colocation, a database with 5000 relations (tables, indexes, etc.) will have at least 5000 tablets, which increases the CPU/disk/network overhead. If most of these relations are small in size, then it's beneficial to use colocated tables.
-
-Colocating various SQL tables puts all of their data into a single tablet, called the colocation tablet.
-Note that all the data in the colocation tablet is still replicated across three nodes (or whatever the replication factor is).
+In workloads that do very little IOPS and have a small data set, the bottleneck shifts from
+CPU/disk/network to the number of tablets one can host per node.
+Since each table by default requires at least one tablet per node, a YugabyteDB cluster with 5000
+relations (tables, indexes) will result in 5000 tablets per node.
+There are practical limitations to the number of tablets that YugabyteDB can handle per node since each tablet
+adds some CPU, disk and network overhead. If most or all of the tables in YugabyteDB cluster are small tables,
+then having separate tablets for each table unnecessarily adds pressure on CPU, network and disk.
+
+To help accomodate such relational tables and workloads, we've added support for colocating SQL tables.
```

Comment:
```suggestion
To help accomodate such relational tables and workloads, you can colocate SQL tables.
```

Reply:
Is this for "SQL tables" only? I thought you said it applies to YCQL too.

Reply:
It's only for SQL.